packages/imperative/src/config/index.ts

```diff
@@ -14,7 +14,7 @@ export * from "./src/ConfigAutoStore";
 export * from "./src/ConfigConstants";
 export * from "./src/ConfigSchema";
 export * from "./src/ConfigBuilder";
-export * as ConfigUtils from "./src/ConfigUtils";
```

Comment:
I think this is the way in which we will have to eventually deal with the Imperative package.
Little by little, and as non-breaking changes. 😋 
Thanks for moving the functions into a class 😋